src/artm/core/processor.cc

```diff
@@ -57,14 +57,16 @@ class RegularizeThetaAgentCollection : public RegularizeThetaAgent {
 
   bool empty() const { return agents_.empty(); }
 
-  virtual void Apply(int item_index, int inner_iter, int topics_size, const float* n_td, float* r_td) const {
+  virtual void Apply(const std::string& item_title, int inner_iter, int topics_size,
+                     const float* n_td, float* r_td) const {
     for (auto& agent : agents_)
-      agent->Apply(item_index, inner_iter, topics_size, n_td, r_td);
+      agent->Apply(item_title, inner_iter, topics_size, n_td, r_td);
   }
 
-  virtual void Apply(int inner_iter, const LocalThetaMatrix<float>& n_td, LocalThetaMatrix<float>* r_td) const {
+  virtual void Apply(int inner_iter, const LocalThetaMatrix<float>& n_td,
+                     LocalThetaMatrix<float>* r_td, const Batch& batch) const {
     for (auto& agent : agents_)
-      agent->Apply(inner_iter, n_td, r_td);
+      agent->Apply(inner_iter, n_td, r_td, batch);
```

Comment:
Maybe it is some OS based problem. I change my OS to Win 10 and tha was the first time I ran tests on it. The test gave different values on my PC and in travis.